### PR TITLE
Added new fields to replace bp - diastolic and systolic

### PR DESCRIPTION
**Pull request by palash-t**

### CODE REVIEW

File: database/tests/conftest.py [hunk 1]
```diff
@@ -84,9 +84,14 @@ def heart_rate():
 	return str(find_one_biometric(db)['heart_rate'])
 
 @pytest.fixture
-def blood_pressure():
+def systolic_blood_pressure():
```
Comment (snelson44) [inline database/tests/conftest.py]: We don't need these blood pressure fixtures anymore

**Review by snelson44 — CHANGES_REQUESTED** [review]
Sorry to be annoying, but we should change the blood pressure field names to sbp and dbp. Also don't run the tests yet. I need to establish a test database that we can test on without blowing away the other teams data.

File: database/tests/conftest.py (hunk 1 above)
Comment (palash-t) [reply database/tests/conftest.py]: Made the required changes.

Comment (snelson44) [reply database/tests/conftest.py]: thanks, I like that a lot better.

**Review by snelson44 — CHANGES_REQUESTED** [review]
You forgot to change the database schema in `init_engine_db.sql`

File: database/tests/test_database.py [hunk 2]
```diff
@@ -109,13 +109,15 @@ def test_find_biometric_by_id(database, patient_id):
...
 	assert str(result_patient_id) == patient_id
-	result_blood_pressure = result['blood_pressure']
+	result_dbp = result['dbp']
+	result_sbp = result['sbp']
 	result_heart_rate = result['heart_rate']
 
 	biometric = find_biometric_by_id(database, result_id)
 	assert biometric['id'] == result_id
 	assert biometric['patient_id'] == result_patient_id
-	assert biometric['blood_pressure'] == result_blood_pressure
```
Comment (snelson44) [inline database/tests/test_database.py:118]: There are still instances of `blood_pressure` in the tests. All these instances should be replaces with sbp and dbp like you did here. Do a find all to find the rest of them

Comment (palash-t) [reply database/tests/test_database.py:118]: Done. Sorry about that.

**Review by snelson44 — APPROVED** [review]

**Pull request merged**